### PR TITLE
Add new wiki to configurator tabs

### PR DESCRIPTION
**Pull request by haslinghuis**
https://betaflight.com/docs/wiki

### CODE REVIEW

**Review by blckmn — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by freasy** [comment]
I dont know if needed :D but:
LQGTM

**Review by chmelevskij — APPROVED** [review]

**Pull request merged**
